src/index.js

```diff
@@ -1,8 +1,6 @@
-//@flow
-import "./polyfill";
+// @flow
 import React, { Component } from "react";
-import { translate } from "react-i18next";
-import { NativeModules } from "react-native";
+import "./polyfill";
```

Comment:
it is important the polyfill is the first thing we import. it might lead to undesired problem otherwise.

Reply:
updated (even if I don't think importing after the `react` import will break something) :)